airflow/providers/dbt/cloud/hooks/dbt.py

```diff
@@ -302,19 +302,22 @@ def _run_and_get_response(
         endpoint: str | None = None,
         payload: str | dict[str, Any] | None = None,
         paginate: bool = False,
+        *,
```

Comment:
Since this is private, why don't we make these all keyword-only args?

Reply:
Yep, indeed a good idea!